dislib/classification/csvm/base.py

```diff
@@ -232,6 +209,25 @@ def score(self, x, y):
 
         return _merge_scores(*partial_scores)
 
+    def _check_initial_parameters(self):
+        gamma = self.gamma
```

Comment:
I use the variable (also self.kernel, self.c and self.tol) multiple times in a single line expression, so I thought it would be more readable like this. This is the full code of the method:
```
    def _check_initial_parameters(self):
        gamma = self.gamma
        assert (gamma == "auto" or type(gamma) == float
                or type(float(gamma)) == float), "Invalid gamma"
        kernel = self.kernel
        assert (kernel is None or kernel in self._name_to_kernel.keys()), \
            "Incorrect kernel value [%s], available kernels are %s" % (
                kernel, self._name_to_kernel.keys())
        c = self.c
        assert (c is None or type(c) == float or type(float(c)) == float), \
            "Incorrect C type [%s], type : %s" % (c, type(c))
        tol = self.tol
        assert (type(tol) == float or type(float(tol)) == float), \
            "Incorrect tol type [%s], type : %s" % (tol, type(tol))
        assert self.cascade_arity > 1, "Cascade arity must be greater than 1"
        assert self.max_iter > 0, "Max iterations must be greater than 0"
        assert type(self.check_convergence) == bool, "Invalid value in " \
                                                     "check_convergence"
```

Reply:
Ok